Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -272,7 +272,7 @@ def fetch_incidents(client: IMAPClient,
         demisto.debug(f"no last_run, using {time_to_fetch_from=}")
 
     # Otherwise use the mail UID
-    uid_to_fetch_from = last_run.get('last_uid', 0)
+    uid_to_fetch_from = int(last_run.get('last_uid', 0))
```

Comment:
Consider adding logs, and using arg_to_number for better errors if they are raised.

Reply:
There are enough logs from my perspective.

Changed to arg_to_number.